CMakeLists.txt

```diff
@@ -472,7 +472,7 @@ target_link_libraries(noisepage_shared PUBLIC noisepage_objlib)             # Co
 
 add_executable(noisepage src/main/terrier.cpp)
 target_link_libraries(noisepage noisepage_static)
-set_target_properties(noisepage PROPERTIES CXX_EXTENSIONS OFF)
+set_target_properties(noisepage PROPERTIES CXX_EXTENSIONS OFF ENABLE_EXPORTS ON)
```

Comment:
I have no idea why ENABLE_EXPORT is necessary for the LLVMEngine's TPLMemoryManager to find its symbols. Any explanations I can throw into CMakeLists.txt would be greatly appreciated.